app/src/routes/definitions.json

```diff
@@ -50,7 +50,11 @@
         "show": "/core/streams/:id",
         "new": "/core/streams/new",
         "public": {
-            "show": "<platform>/core/streams/:id"
+            "show": "<streamr>/streams/:id"
+        },
+        "root": {
```

Comment:
Can you think of better naming scheme here?

Reply:
Maybe all under `public`?

```
"public": {
    "stream": "<streamr>/streams/:id",
    "show": "/streams/:id",
    "preview": "/streams/:id/preview"
}
```